vital/types/geodesy.h

```diff
@@ -56,16 +57,52 @@ namespace vital {
  * \see https://en.wikipedia.org/wiki/Spatial_reference_system,
  *      http://www.epsg.org/, https://epsg-registry.org/
  */
-enum class SRID : int
+namespace SRID
 {
-  lat_lon_NAD83 = 4269,
```

Comment:
Yes. In practice, there are two ways to describe a CRS; provide a complete description as e.g. [Well-Known Text](https://en.wikipedia.org/wiki/Well-known_text), or refer to an entry in the EPSG database (which can be done by SRID — examples: [1](http://spatialreference.org/ref/epsg/4269/), [2](https://epsg.io/4269)). See also [the comment](https://github.com/Kitware/kwiver/blob/master/vital/types/geodesy.h#L46-L57) just above these.

Carrying around a complete CRS specification for such a low level type would increase the storage requirements for the type significantly, and to almost no purpose, as it is usually sufficient to specify an EPSG-defined CRS. Note that this is also how ViViA's geodetic types work (see [vgGeoSystem](https://github.com/Kitware/vivia/blob/master/Libraries/VgCommon/vgGeoTypes.h#L13-L17) and consuming classes).